glow/src/triangle/gradient.rs

```diff
@@ -0,0 +1,189 @@
+use crate::program::Version;
+use crate::triangle::{simple_triangle_program, update_transform};
+use glow::{Context, HasContext, NativeProgram};
+use iced_graphics::gradient::Gradient;
+use iced_graphics::widget::canvas::gradient::Linear;
+use iced_graphics::Transformation;
+
+#[derive(Debug)]
+pub(super) struct GradientProgram {
```

Comment:
Nit: Do these need to be `super`? I'm not sure `GradientProgram` should be part of the public API of `iced_glow`, and if it's not already... Then it's fine to just leave it public for the whole crate.